services/account-service-api/AccountService.kt

```diff
@@ -0,0 +1,164 @@
+package classtime.service.account
```

Comment:
It was the fastest and easiest way :smile_cat: And it seems a pretty clean option.
- I have tried `dokka` and `javaDoc` task. I like neither of them... Some other options generation interfaces look awful.
- Yeah, protobuf service definition in general sounds nice. If not the limitations you mention :(

Reply:
Actually, `dokka` generated file looks pretty good. Also It could generate both .html and .md versions. The only curve edges are:
- There is no option to include only a single interface in a generation. So it generates also protobuf classes. However, I come up with a hack to delete `gen` folder before dokka generation.
- It loose `Service.NAME` value. dokka cannot show strings values ([See this issue](https://github.com/Kotlin/dokka/issues/2408)). But, maybe it's fine to adjust this only place manually by some commands.

Reply:
I have created a separate branch, where you could take a look on dokka example. In this branch, it would create too much noise.

https://github.com/gopollock/public-api/tree/dokka-example/classtime/service/account

Reply:
Yeah, looks okay. I do not care about the NAME constant (it should always match with the prefix of the class anyways).

But now it says "Error unknown class" for all protos. We not create documentation for protos too?

Reply:
Ahhh, I haven't seen this side-effect :( Index files become noisy with protos included.
Anyway, I will create PR from example branch and we could discuss it there.

Maybe it is fine for you or we could fix it with simple file move / rename / delete.